### PR TITLE
Leader-Board:Final touch

### PR DESCRIPTION
**Pull request by amanuelbelete95**
## List of Completed Activity

- Added basic styling
- Updated the project looks
- Kept up the layout of the wireframe

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -31,7 +40,18 @@ header {
...
+.score {
+  padding: 10px;
+  width: 200px;
+}
+
+.score:nth-child(odd) {
+  background-color: rgba(128, 128, 128, 0.39);
 }
 
 .form {
```
Comment (OLIPLICHE) [inline src/style.css:56]: - [ ] **Styling:**
- I would advise you to properly style your App and its layout should look like the wireframe below. This is part of the project requirement.
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @amanuelbelete95, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit has a green check mark (✅)) 👏
- Commit messages are meaningful 👏 👏

## Required Changes ♻️
- I would advise that your README file looks professional and contains all that's needed for running your app with the API(s) it consumes. It should have instructions about where and how to generate any necessary keys and credentials, and how to set, store, and generate them.
- 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @amanuelbelete95 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

You have submitted a project that has linter tests falling. Please make sure all linters are set up and passing before your next review request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/style.css [hunk 2]
```diff
@@ -47,5 +66,8 @@ input {
 }
 
 .submit {
-  width: 50%;
+  /* width: 50%; */
```
Comment (huseyinbiyik) [inline src/style.css:69]: [OPTIONAL] Please don't leave any commented code in your codebase. In this way, you can keep your codebase clean, organized and professional.

File: src/style.css (hunk 1 above)
Comment (huseyinbiyik) [reply src/style.css:56]: As the previous reviewer requested, please follow the wireframe. Currently, it looks like below.
Screenshot 2023-03-15 at 9 41 42 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @amanuelbelete95,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Linter set up well and checks pass 🟢
- Git flow followed 👌
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
[OPTIONAL] Please ensure your live demo links works as intended. Currently, it gives 404. Ifı you don't want to deploy your app, you can remove that section.
Screenshot 2023-03-15 at 9 41 32 PM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @amanuelbelete95 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Used Git flow ✔️
## Optional suggestions
- [OPTIONAL] Please ensure your live demo links works as intended. Currently, it gives 404. If you don't want to deploy your app, you can remove that section.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**